test/ui-test/testSuites/suite_communities/tst_communityMessageFlows/test.feature

```diff
@@ -139,3 +139,20 @@ Feature: Status Desktop community messages
         | displayName        |
         | notExistingAccount |
         | asdfgNoNo          |
+
+    Scenario: The user can send an emoji as a message
+        When the user sends the emoji "heart_eyes" as a message
+        Then the last chat message contains "😍"
+
+    Scenario Outline: The user can send an emoji in a message
+        When the user sends the emoji "sunglasses" with message "<message>"
+        Then the last chat message contains "😎"
+        And the last chat message contains "<message>"
+        Examples:
+         | message          |
+         | wow I'm so cool  |
+
+    Scenario: The user can type message with emoji autoreplace
+        When the user sends a chat message "Hello :)"
+        Then the last chat message contains "🙂"
+        And the last chat message contains "Hello"
```

Comment:
Can we do validation that the message contains the correct string (text + emoji) rather than emoji and text separately?

Reply:
iirc, the reason it was done like this originally is because the emoji inside the message is actually an HTML image. The emoji as utf is only present in the `alt`. That's why checking for the individual emoji works, but checking for the text + emoji is hard.